src/Ads/index.js

```diff
@@ -16,223 +16,77 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-import Log from '../Log'
-import { mediaUrl } from '../VideoPlayer'
-
-let consumer
-
-let getAds = () => {
-  // todo: enable some default ads during development, maybe from the settings.json
-  return Promise.resolve({
-    prerolls: [],
-    midrolls: [],
-    postrolls: [],
-  })
-}
+import AdsPlayer from './player'
+import { initAds as initAdsPlayer } from './player'
 
 export const initAds = config => {
   if (config.getAds) {
-    getAds = config.getAds
+    initAdsPlayer(config)
   }
 }
 
-const state = {
-  active: false,
+let getAdvertisingId = function() {
```

Comment:
we also need to add methods:
`deviceAdAttributes()` (returns object)
`appStoreId()` (returns string)